libraries/lib-wave-track/WaveClip.cpp

```diff
@@ -1427,14 +1436,12 @@ void WaveClip::TrimRight(double deltaTime)
 
 void WaveClip::TrimLeftTo(double to)
 {
-   mTrimLeft =
-      std::clamp(to, SnapToTrackSample(mSequenceOffset), GetPlayEndTime()) -
-      mSequenceOffset;
+   mTrimLeft = std::clamp(to, mSequenceOffset, GetPlayEndTime()) - mSequenceOffset;
```

Comment:
Maybe it's okay this way, but I'd round `to`. Other places in the code (`WaveClipAdjustBorderHandle`, `WaveTrack::SplitAt`) round the argument before calling the method, only `::ApplyStretchRatio` doesn't. It might cause problems and I don't see a use case for trimming to places between samples.